tests/jenkins/jobs/build-and-assemble.groovy

```diff
@@ -0,0 +1,180 @@
+// similar jenkins/opensearch/Jenkinsfile but echo instead of sh for testing
```

Comment:
its this automatically invoked as a test? I didn't see it in the check for gradle on this PR

Reply:
Just saw that this was already called out